src/newSeed/newSeed.html

```diff
@@ -1,12 +1,22 @@
 <template>
+  <require
+    from="../resources/elements/circledNumber/circledNumber.html"
+  ></require>
   <div class="newSeedContainer">
     <div class="currentStageSidebar">
       <template repeat.for="stage of router.navigation">
         <div
           class="currentstage ${stage.isActive ? 'active' : ''}"
           click.delegate="setStage(stage.config.route)"
         >
-          ${$index+1} ${stage.config.title}
+          <div class="stagesNumber">
+            <circled-number number.bind="$index + 1"></circled-number>
+            <div>${stage.config.title}</div>
+          </div>
+          <div
```

Comment:
what is this div for?

Reply:
Hold the title, use it to also flex the display of the stage numbers

Reply:
it doesn't hold the title.  it is empty.

Reply:
Ha, i see the div you mean. It is for the lines connecting the stage number together.

Reply:
If you run the app, you would see the line. If you intend to implement this differently, let me knoe

Reply:
Here is a screenshot
<img width="416" alt="Screenshot 2021-04-15 at 23 16 59" src="https://user-images.githubusercontent.com/19796652/114945309-fdea0500-9e40-11eb-8c72-c67edaf31bd9.png">